opencsp/app/sofast/lib/DistanceOpticScreen.py

```diff
@@ -30,14 +30,14 @@ def save_to_hdf(self, file: str, prefix: str) -> None:
             Prefix to append to folder path within HDF file (folders must be separated by "/").
```

Comment:
It is unclear whether the intention is for the prefix parameter to be a folder (ending in '/') or if it is just supposed to be the beginning of the name. I'd suggest updating the documentation and/or parameter name with the intended usage. Maybe something like:

```python
def save_to_hdf(self, file: str, prefix: str = '') -> None:
    """Saves data to given file. Data is stored as: PREFIX + Folder/Field_1
    Parameters
    ----------
    file : str
        HDF file to save to
    prefix : str, optional
        Prefix to prepend to the folder path within the HDF file. Suggested use
        is as a prefix to the folder name such as "202407_BCS_", so that folders
        can still be easily machine identified. Note that folders must be
        separated by "/", so if prefix is a folder name, then it should end in
        "/". Default is empty string ''.
    """
```

Note also that I changed "append" to "prepend".